src/v2/common/libraries/VaultLib.sol

```diff
@@ -133,9 +134,12 @@ library VaultLib {
             0
         );
 
+        uint256 sharesAfter = ERC20(_config.receiptToken).balanceOf(address(vault));
+        uint256 shares = sharesBefore - sharesAfter;
+
         uint256 withdrawn_ = abi.decode(withdrawnRaw, (uint256));
```

Comment:
chose not to return `amount` because we get `withdrawn` as a return value from the `manage()` call